clvm/operators.py

```diff
@@ -111,6 +111,10 @@ def default_unknown_op(op: bytes, args: CLVMObject) -> Tuple[int, CLVMObject]:
 
     cost_function = (op[-1] & 0b11000000) >> 6
     # the multiplier cannot be 0. it starts at 1
+
+    if len(op) > 5:
+        raise EvalError("invalid operator", args.to(op))
```

Comment:
the new requirements is that the cost multiplier may not be more than 4 bytes wide. It's still interpreted as an unsigned integer. This restriction was kind of implied before, by ensuring the total cost of the operation did not exceed `uint32_max`.

Reply:
Can you add the comment that it's still interpreted as an unsigned integer?
Maybe that is discussed during cost computation.

Reply:
Can you add the comment that it's still interpreted as an unsigned integer?
Maybe that is discussed during cost computation.